File: output/anthropic_claude_sonnet_4_6/zero/python_noimg/Cobra coral.py

a, b, c, d = map(int, input().split())

# Verdadeira: BVBPBVBP -> padrão de 4: posições 0,1,2,3 = B,V,B,P
# então a==c e a!=b e a!=d e b!=d
# Falsa: BVPBVPBVP -> padrão de 4: posições 0,1,2,3 = B,V,P,B
# então a==d e a!=b e a!=c e b!=c

if a == c and a != b and a != d and b != d:
    print("V")
else:
    print("F")

File: output/csbc_semish_2026/z_ai_glm_5/few/cpp/Cobra coral.cpp

#include <bits/stdc++.h>

using namespace std;

int main() {
    ios::sync_with_stdio(0);
    cin.tie(0);

    int a, b, c, d;
    if (cin >> a >> b >> c >> d) {
        // A falsa coral tem padrão BVPBVP... (período 3).
        // Em qualquer janela de 4 elementos (x_i, x_{i+1}, x_{i+2}, x_{i+3}),
        // como o período é 3, temos x_i = x_{i+3}.
        // Logo, se o primeiro e o último número forem iguais, é uma Falsa Coral (F).
        
        // A coral verdadeira tem padrão BVBPBVBP... (período 4).
        // O ciclo é B, V, B, P. Os elementos B e P são distintos.
        // Em qualquer janela de 4, o primeiro e o último elementos são distintos.
        // (Ex: B,V,B,P -> B!=P; V,B,P,B -> V!=B; B,P,B,V -> B!=V; P,B,V,B -> P!=B).
        // Logo, se o primeiro e o último número forem diferentes, é uma Coral Verdadeira (V).
        
        if (a == d) {
            cout << "F" << endl;
        } else {
            cout << "V" << endl;
        }
    }
    return 0;
}
